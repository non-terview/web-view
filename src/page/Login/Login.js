import LoginForm from "../../components/LoginForm/LoginForm";

function Login() {
    return (
        <LoginForm>
        </LoginForm>
    );
}

export default Login;