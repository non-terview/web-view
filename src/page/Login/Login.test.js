import Login from './Login';
import renderer from "react-test-renderer";
import React from "react";


test('Login changes the class when hovered', () => {
        const component = renderer.create(
            <Login/>,
        );

        let form = component.toJSON();

        expect(form).toMatchSnapshot();
    }
)