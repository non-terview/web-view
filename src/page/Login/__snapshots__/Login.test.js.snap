// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login changes the class when hovered 1`] = `
<div
  className="sc-gsTEea evOHZW"
>
  <div>
    <div>
      <input
        className="sc-bdfBQB erSyrG"
        onChange={[Function]}
        placeholder="email"
        type="text"
        value=""
      />
      <h1>
        
      </h1>
    </div>
    <div>
      <input
        className="sc-bdfBQB erSyrG"
        onChange={[Function]}
        placeholder="password"
        type="password"
        value=""
      />
      <h1>
        
      </h1>
    </div>
  </div>
  <input
    className="sc-dlfnuX logZOV"
    onClick={[Function]}
    type="button"
    value="회사 로그인으로 전환"
  />
</div>
`;
